Description: Fix compilation on architectures other than amd64 and i386
 This fixes a compilation error in the ucontext case of my_context.c.
Author: Daniel Schepler <schepler@debian.org>
Forwarded: no

Index: mariadb-client-lgpl/libmariadb/my_context.c
===================================================================
--- mariadb-client-lgpl.orig/libmariadb/my_context.c
+++ mariadb-client-lgpl/libmariadb/my_context.c
@@ -126,7 +126,7 @@ my_context_yield(struct my_context *c)
 int
 my_context_init(struct my_context *c, size_t stack_size)
 {
-#if sizeof(char *) > SIZEOF_INT*2
+#if SIZEOF_CHARP > SIZEOF_INT*2
 #error Error: Unable to store pointer in 2 ints on this architecture
 #endif
   bzero(c, sizeof(*c));
